src/CacheInterface.php

```diff
@@ -70,7 +70,7 @@ public function getMultiple($keys, $default = null);
      * Persists a set of key => value pairs in the cache, with an optional TTL.
      *
      * @param iterable              $values A list of key => value pairs for a multiple-set operation.
-     * @param null|int|DateInterval $ttl    Optional. The TTL value of this item. If no value is sent and
```

Comment:
Because here whitespace should not be placed - https://phpdoc.org/docs/latest/guides/types.html#multiple-types-combined